.stylelintrc.json

```diff
@@ -3,8 +3,8 @@
   "rules": {
     "max-nesting-depth": [1, { "ignore": ["pseudo-classes"] }],
     "selector-class-pattern": "^-?[a-z]+(?:--?[a-z]+)*$",
-    "scss/dollar-variable-pattern": "^[a-z]+(?:-[a-z]+)*$",
-    "scss/at-function-pattern": "^[a-z]+(?:-[a-z]+)*$",
+    "scss/dollar-variable-pattern": "^_?[a-z]+(?:-[a-z]+)*$",
```

Comment:
Maybe extract this (private scss variables) and the `space` change to another PR so we can quickly merge it (with nice commit messages)?

Reply:
Sure:

- https://github.com/onfido/castor/pull/155
- https://github.com/onfido/castor/pull/156